course04/homework00/shankarvasudevan/sortedarray/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.5.1)
```

Comment:
Thats fine but in general I'd say if you dont use anything that has been added into this very specific version, why putting a requirement on it? It might block people using your project.